metaci/build/models.py

```diff
@@ -333,13 +333,21 @@ def run(self):
             # Look up or spin up the org
             org_config = self.get_org(project_config)
             if (
-                self.org and self.org.name and self.org.name.lower() == "packaging"
-            ):  # Calling for any actions taken against packaging org
-                send_start_webhook(
-                    self.release,
-                    self.plan.role,
-                    self.org.configuration_item,
-                )
+                self.plan.change_traffic_control
+            ):
+                try:
+                    send_start_webhook(
+                        self.release,
+                        self.plan.role,
+                        self.org.configuration_item,
+                    )
+                except Exception as err:
+                    message = f"Error while sending implementation start step webhook: {err}"
+                    self.logger.error(message)
+                    set_build_info(
+                        build, status="error", exception=message, time_end=timezone.now()
+                    )
+                    return
```

Comment:
We don't need this exception handler; it will be handled the same way by the block starting on line 344.

Also you can simply write `if self.plan.change_traffic_control:` and not split it over 3 lines